internal/pkg/deploy/cloudformation/app.go

```diff
@@ -456,9 +456,12 @@ func (cf CloudFormation) cleanUpRegionalResources(app *config.Application, regio
 	if err := s3.EmptyBucket(resources.S3Bucket); err != nil {
 		return err
 	}
+	ecrRepoName := func(app, workload string) string {
+		return fmt.Sprintf("%s/%s", app, workload)
+	}
 	ecr := cf.regionalECRClient(region)
-	for repo := range resources.RepositoryURLs {
-		if err := ecr.ClearRepository(repo); err != nil {
+	for svcName := range resources.RepositoryURLs {
```

Comment:
nit: because the field is named "RepositoryURLs", I was surprised to see `svcName` - consider the following to make it obvious that its a map:
```suggestion
	for svcName, _ := range resources.RepositoryURLs {
```

Also, do we need to change the unit tests?

Reply:
done bossu!

Reply:
Why do we manually construct the repo name as opposed to using values of the map? Is it more difficult to parse the repo name out?

Reply:
We could parse from the URL but it is indeed not as straightforward as constructing the repo name. I feel like either way it's like potato poTAto 👀